tiles/dist/main.js

```diff
@@ -90,7 +90,7 @@ eval("var require;var require;(function(f){if(true){module.exports=f()}else { va
"fill-color\": \"#ff00ff\" //my violet water\n        }\n    });\n    console.log(map);\n}\nfunction setMaputnikStyle(){\n    map.setStyle('styles/my_first_maputnik_style.json');\n}\n\nfunction setMapboxStyle(){\n    map.setStyle('mapbox://styles/nhofer/cje5h47heffe42rt50nvm7zyn');\n}\n\n//piece of wms in new york\nfunction addWMS() {\n    // would like to check the id of the layer we have got to put before\n    // but since there is too many changesgit \n    // var idOfFirstLayer = map.getStyle().layers[0].id   \n        map.addLayer({\n        'id': 'wms-test-layer',\n        'type': 'raster',\n        'source': {\n            'type': 'raster',\n            'tiles': [\n                'https://geodata.state.nj.us/imagerywms/Natural2015?bbox={bbox-epsg-3857}&format=image/png&service=WMS&version=1.1.1&request=GetMap&srs=EPSG:3857&width=256&height=256&layers=Natural2015'\n            ],\n            'tileSize': 256\n        },\n        'paint': {}\n    });\n    map.setPaintProperty('wms-test-layer', 'raster-opacity', 0.5);\n}\n\nfunction setSchweizMobile(){\n   map.setStyle(SwissTopoStyle); \n}\n\nfunction switchLayer(layer) {\n    var layerId = layer.target.id;\n    switch(layerId) {\n        case 'basic':\n        case 'streets':    \n            map.setStyle('mapbox://styles/mapbox/' + layerId + '-v9');\n            break;\n        case 'simple-custom-maputnik':\n            setMaputnikStyle();\n            break;\n        case 'simple-custom-mapbox':\n            setMapboxStyle();\n            break;\n        case 'schweiz-mobile-like' :\n            setSchweizMobile();\n    }\n}\n\n\n\n//swisstopodata\nfunction addWMTS() {\nmap.addSource(\"bauzonen\", {\n        \"attribution\": \"&copy; swisstopo\",\n        \"bounds\": [\n            5.140242,\n            45.398181,\n            11.47757,\n            48.230651\n        ],\n        \"maxzoom\": 18,\n        \"minzoom\": 0,\n        \"type\": \"raster\",\n        \"tileSize\": 256,\n        \"tiles\": [\n            \"https://wmts.geo.admin.ch/1.0.0/ch.are.bauzonen/default/current/3857/{z}/{x}/{y}.png\"\n        ]\n    });\n\n    map.addLayer({\n        \"id\": \"bauzonen\",\n        \"type\": \"raster\",\n        \"source\": \"bauzonen\",\n        \"minzoom\": 0,\n        \"maxzoom\": 18,\n        \"paint\": {\n            \"raster-opacity\": 0.3,\n        }\n});\n}\n//bind functions to menu\n\nvar layerList = document.getElementById('menu');\nvar inputs = layerList.getElementsByTagName('input');\n\nfor (var i = 0; i < inputs.length; i++) {\n    inputs[i].onclick = switchLayer;\n}\n//bind buttons to functions\ndocument.getElementById(\"addWMS\").addEventListener(\"click\", addWMS);\ndocument.getElementById(\"addWMTS\").addEventListener(\"click\", addWMTS);\ndocument.getElementById(\"findMyHse\").addEventListener(\"click\", findMyHouse);\ndocument.getElementById(\"addThirdPartyLr\").addEventListener(\"click\", \naddThirdPartyLayer);\n\n\n\n\n\n\n\n//# sourceURL=webpack:///./src/index.js?");
```

Comment:
Again not really a problem for a small project, but for bigger amounts of code it's usually good not to include the built file in the repository history in order to keep it small and clean.